cypress/e2e/laboratory-environment-variables.cy.ts

```
@@ -0,0 +1,84 @@
+    buttonSubmitCy: 'preflight-modal-submit',
+  },
+};
+
+// todo: instead of copying this, import it from core utility lib.
+export const environmentVariablesStorageKey = {
+  // todo: optional target effectively gives this the possibility of being silently global
+  // which feels subtle and thus likely to introduce hard to trace defects. Should we abort instead?
+  scoped: (targetId?: string) =>
+    `hive/targetId:${targetId ?? '__null__'}/laboratory/environment-variables`,
+  global: 'hive:laboratory:environment',
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)